### DISCUSSION TITLE
빅데이터 분석 자료

### DISCUSSION THREAD

**Discussion by Starlight258** [Starlight258/BigDataAnalysis#4] [General]
분석 자료 정리

**Comment by H-sooyeon** [comment]
2022년-12월-조사결과 :: 
99p 코로나19에 대한 두려움
109p 불안(GAD-7)
121p 불안 위험군
137p 우울(PHQ-9)
149P 우울 위험군
177P 외상 후 스트레스 장애(PTSD) 위험군

**Comment by H-sooyeon** [comment]
주제를 코로나19로 크게 잡은 후
부정, 긍정으로 분류

분석 방법: 상관분석
긍정: 코로나 마스크 착용으로 인한 감기 등 호흡기 질병 발생 감소
부정: 코로나블루 (정신적)

**Comment by H-sooyeon** [comment]
**분석 방법**
긍정: 시계열 분석
부정: 시계열 분석, 상관분석

**부정 시각화**
히트맵 (명령어 사용 가능) -> 상관관계가 강할 수록 진한 색으로 그래프 표현
선 그래프

**긍정 시각화**
선 그래프

**Comment by Starlight258** [comment]
[2022년-12월-조사결과-COVID-19-국민정신건강실태조사 (1).pdf](https://github.com/Starlight258/BigDataAnalysis/files/11494923/2022.-12.-.-COVID-19-.1.pdf)

**Comment by Starlight258** [comment]
국민건강보험공단 질병예측정보
https://www.data.go.kr/data/15083145/fileData.do

**Comment by Starlight258** [comment]
시계열 그래프
https://whatisthenext.tistory.com/108
https://rfriend.tistory.com/73

**Comment by Starlight258** [comment]
1. 긍정 시계열 그래프
광주 - 감기, 피부염, 천식
가로축: 2020년 3월~22년 10월까지 달 별
세로축: 감기, 피부염, 천식 발생건수

2. 부정 데이터
- 우울, 불안 시계열 그래프
가로축: 날짜
세로축:카테고리 

- 상관성 분석
코로나 감염자수 - 우울(PHQ),불안(GAD), 코로나19에 대한 두려움 평균값 
감염자수에 대한 상관성 각각 작성

**Comment by Starlight258** [comment]
**수연**
- 데이터 2개 : 감기, 피부염  

**명지**
- 데이터 1개 : 천식
- 그래프

**Comment by Starlight258** [comment]
**시계열 그래프**

**asthma 그래프**
![timegraph_asthma](https://github.com/Starlight258/BigDataAnalysis/assets/78211281/224d35da-c459-4178-a8e0-d7c0ba2ac876)

**cough 그래프**
![timegraph_cough](https://github.com/Starlight258/BigDataAnalysis/assets/78211281/3d95eafa-2bc2-42a8-bce9-40529b156fdd)

**dermatitis 그래프**
![timegraph_dermatitis](https://github.com/Starlight258/BigDataAnalysis/assets/78211281/a2448f44-be82-4c24-8562-a414728b5a70)

**Comment by H-sooyeon** [comment]
코로나 감염자 수 csv 파일 출처

KDX 한국데이터거래소
https://kdx.kr/data/view/3415
